packages/server/src/server.ts

```diff
@@ -1,50 +1,40 @@
-import { HttpHeader, HttpHeaderRequestId, LambdaAlbRequest } from '@linzjs/lambda';
-import { handleRequest } from '@basemaps/lambda-tiler';
-import * as ulid from 'ulid';
+import { handler } from '@basemaps/lambda-tiler';
+import { ALBEvent, ALBResult, APIGatewayProxyResultV2, CloudFrontRequestResult, Context } from 'aws-lambda';
+import ulid from 'ulid';
 import express from 'express';
-import { LogConfig } from '@basemaps/shared';
-import { Context } from 'aws-lambda';
+import { lf } from '@linzjs/lambda';
 
 export const BasemapsServer = express();
 
-BasemapsServer.get('/v1/*', async (req: express.Request, res: express.Response) => {
-  const startTime = Date.now();
-  const requestId = ulid.ulid();
-  const logger = LogConfig.get().child({ id: requestId });
-  const ctx = new LambdaAlbRequest(
-    {
-      httpMethod: 'get',
-      path: req.path,
-      queryStringParameters: req.query,
-    } as any,
-    {} as Context,
-    logger,
-  );
+function isAlbResult(r: ALBResult | CloudFrontRequestResult | APIGatewayProxyResultV2): r is ALBResult {
+  if (typeof r !== 'object') return false;
+  if (r == null) return false;
+  return 'statusCode' in r;
+}
+
+const instanceId = ulid.ulid();
 
-  if (ctx.header(HttpHeaderRequestId.RequestId) == null) {
-    ctx.headers.set(HttpHeaderRequestId.RequestId, 'c' + requestId);
-  }
-  const ifNoneMatch = req.header(HttpHeader.IfNoneMatch);
-  if (ifNoneMatch != null && !Array.isArray(ifNoneMatch)) {
-    ctx.headers.set(HttpHeader.IfNoneMatch.toLowerCase(), ifNoneMatch);
-  }
+BasemapsServer.get('/v1/*', async (req: express.Request, res: express.Response) => {
+  const event: ALBEvent = {
+    httpMethod: 'GET',
+    requestContext: { elb: { targetGroupArn: 'arn:fake' } },
```

Comment:
fake Arn?

Reply:
it needed a 'string' for type completeness, doesn't matter what the string is as we dont use it anywhere

Reply:
Weird, this cannot be optional?

Reply:
Nope, this is a AWS ALB event type, so all ALB events should have a arn value there.